<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_null_TipDialog</class>
 <widget class="QDialog" name="dialog_null_TipDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_tipDialog_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提示</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_tipDialog_shutdown">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_tipDialog_content">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>配置已修改，确定要保存吗？</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tipDialog_sure">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tipDialog_cancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
